### DISCUSSION TITLE
API Calls for Transit Routes and Stops Slow

### DISCUSSION THREAD

**Discussion by dinwun450** [transitland/transitland#424] [Q&A] [answered]
Why does the response time of transit.land v2 APIs in fetching routes and stops info going so slow? I couldn't load the routes after a long time and got a 504 error.

**Comment by dinwun450** [comment]
Is there such strange behavior occurring in the v2 REST API or is the fetching of the transit route information down?

**Comment by drewda** [comment, marked as answer]
Since Thursday, April 18, certain API queries have had degraded performance.

The Interline team has reached out to users on the professional and enterprise plans using those API endpoints, and has prioritized maintenance work to get paying customers' queries back up to normal performance.

We are almost complete with the maintenance and repairs, and expect performance for free plan API users to be back to normal already and website users to be back to normal soon.

We appreciate everyone's patience.
